integration/main_test.go

```diff
@@ -208,6 +208,13 @@ func WithPodLabels(kvs map[string]string) PodSandboxOpts {
 
 // PodSandboxConfig generates a pod sandbox config for test.
 func PodSandboxConfig(name, ns string, opts ...PodSandboxOpts) *runtime.PodSandboxConfig {
+	var cgroupParent string
+	runtimeConfig, _ := runtimeService.RuntimeConfig(&runtime.RuntimeConfigRequest{})
+	if runtimeConfig != nil &&
+		runtimeConfig.GetLinux() != nil &&
+		runtimeConfig.GetLinux().GetCgroupDriver() == runtime.CgroupDriver_SYSTEMD {
```

Comment:
You don't need to check nil pointer here. we can call `runtimeConfig.GetLinux().GetCgroupDriver() == runtime.CgroupDriver_SYSTEMD` directly. And why don't check error here?

Reply:
don't check error reason: I hope to use downgrade processing here. If the request for the `RuntimeConfig` api fails, the previous logic will be used.